.changelog/43825.txt

```diff
@@ -0,0 +1,7 @@
+```release-note:new-data-source
+aws_odb_gi_versions_list
```

Comment:
can we remove `_list` from the end of the CHANGELOG entries; to match the resource names